docsrc/tools/Program.fs

```diff
@@ -1,191 +0,0 @@
-﻿// Learn more about F# at http://fsharp.org
```

Comment:
the code here is needed (as seen in the other PR), but could be turned into a FAKE script